convo_utility.py

```diff
@@ -1,5 +1,5 @@
 from conversation import *
-from chirps_utility import *
+import chirps_utility
```

Comment:
Still have no idea why the other way refused to work in this particular file.  I'll run it by @chortlehoort